AppSharedPackageContainer Documentation (#2915)
MicrosoftDocs/windows-powershell-docs: docset/winserver2022-ps/appx/Remove-AppSharedPackageContainer.md

﻿---
external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-Help.xml
Module Name: appx

> online version: https://docs.microsoft.com/powershell/module/appx/move-appxpackage?view=windowsserver2022-ps&wt.mc_id=ps-gethelp

Reviewer: ```suggestion
online version: https://docs.microsoft.com/powershell/module/appx/remove-appsharedpackagecontainer?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
```

schema: 2.0.0
---

# Remove-AppSharedPackageContainer

## SYNOPSIS
Removes the shared package container.

## SYNTAX

```
Remove-AppSharedPackageContainer [-Name] <String> [-ForceApplicationShutdown] [-AllUsers] [<CommonParameters>]
```

## DESCRIPTION 

> The Remove-AppSharedPackageContainer cmdlet removes the shared package container definiton for the particular user.

Reviewer: Cmdlet names should be in backticks in a paragraph. But you don't need the cmdlet name here because it is redundant.
```suggestion
The cmdlet removes the shared package container definition for the particular user.
```

## EXAMPLES

### Example 1
```powershell
PS C:\> Remove-AppSharedPackageContainer -Name ContosoTestContainer
```

> This command removes the shared package container definiton with the name ContosoTestContainer

Reviewer: ```suggestion
This command removes the shared package container definition with the name ContosoTestContainer.
```

## PARAMETERS

### -AllUsers

> Remove matching packages that are deployed to any user

Reviewer: ```suggestion
Remove matching packages that are deployed to any user.
```

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -ForceApplicationShutdown

> Closes all packages in the Shared Package Container

Reviewer: ```suggestion
Closes all packages in the Shared Package Container.
```

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Name

> Name of the container

Reviewer: ```suggestion
The name of the container.
```
